rhscl-postgresql-apb/README.md

```diff
@@ -0,0 +1,25 @@
+rhscl-postgresql-apb
+======================
+
+An apb for deploying [PostgreSQL](https://www.postgresql.org).
+
+## What it does
+* Deploys 1 postgresql container.
+
+## Requirements
+
+## Parameters
+* NAMESPACE: Optional, default 'postgresql-demo-apb', Namespace to deploy the cluster in.
+* POSTGRESQL_DATABASE, default 'admin', Postgresql database name.
+* POSTGRESQL_PASSWORD, default is a randomly generated string, Postgresql database password.
```

Comment:
OK so ```--extra-vars``` will be for the command ```ansible-playbook ...```?  In that case, you could also do ```-e @/etc/my.cnf``` with ansible to get the same affect as ```--env-file``` from the docker cli.  Either way, I'm not arguing one way or the other. Just trying to add more information.